sql/core/src/test/resources/sql-tests/results/url-functions.sql.out

```diff
@@ -95,6 +95,22 @@ struct<url_decode(https%3A%2F%2Fspark.apache.org):string>
 https://spark.apache.org
 
 
+-- !query
```

Comment:
This test is broken with JDK 11 and 17:

```
[info] - url-functions.sql *** FAILED *** (313 milliseconds)
[info]   url-functions.sql
[info]   Expected "...scape (%) pattern - [For input string]: \"2s\""
[info]     }
[info]   }", but got "...scape (%) pattern - [Error at index 1 in]: \"2s\""
[info]     }
[info]   }" Result did not match for query #12
[info]   select url_decode('http%3A%2F%2spark.apache.org') (SQLQueryTestSuite.scala:476)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at org.scalatest.Assertions.newAssertionFailedException(Assertions.scala:472)
[info]   at org.scalatest.Assertions.newAssertionFailedException$(Assertions.scala:471)
[info]   at org.scalatest.funsuite.AnyFunSuite.newAssertionFailedException(AnyFunSuite.scala:1563)
[info]   at org.scalatest.Assertions.assertResult(Assertions.scala:847)
[info]   at org.scalatest.Assertions.assertResult$(Assertions.scala:842)
[info]   at org.scalatest.funsuite.AnyFunSuite.assertResult(AnyFunSuite.scala:1563)
```

https://github.com/apache/spark/runs/8040214816?check_suite_focus=true
https://github.com/apache/spark/runs/8045261892?check_suite_focus=true

Reverting for now.